docs/fsharp/language-reference/import-declarations-the-open-keyword.md

```diff
@@ -62,6 +62,25 @@ open type M.DU
 
 printfn "%A" A
 ```
+## Open from root path only with `global` keyword
+Nested modules like 
+```fsharp
+module A =
+    module B =
+        ...
+```
+can be opend via
+```fsharp
+open A // opens A
+open B // opens A.B
+```
+To open **only** fully qualified modules or namspaces prefix them with the `global` keyword:
```

Comment:
global is not a keyword, but a specifier which tells compiler to search from the root.

Reply:
fixed